src/index.ts

```diff
@@ -70,14 +70,32 @@ const downloadTraceeCommercial = async () => {
   }
 }
 
-const executeTraceeInBackground = async (
-  repoPath: string,
-  aquaKey: string,
-  aquaSecret: string,
-  accessToken: string
-) => {
-  const command = `bash`
-  await exec(command, ['-c', `./tracee ci start -r "${repoPath}" &`], {
+const generateCommand = (flags: TraceeStartFlags): string => {
+  const traceeCommand = ['./tracee', 'ci', 'start', '-r', `"${flags.repoPath}"`]
+
+  if (flags.verbose && !flags.quiet) {
+    traceeCommand.push('-v')
+  }
+
+  if (flags.quiet) {
+    traceeCommand.push('-q')
+  }
+
+  if (flags.logFile) {
+    traceeCommand.push('--log-file')
+    traceeCommand.push(`"${flags.logFile}"`)
```

Comment:
2 checks:
1. That the file doesn't already exist
2. That the folder does exist